### DISCUSSION TITLE
Is it possible to create an overlay based on an items index in the collection?

### DISCUSSION THREAD

**Discussion by 8bit-echo** [meisnate12/Plex-Meta-Manager#1358] [Ideas]
Example: I have a "top 10 movies on server" collection from Tautulli. Would it be possible to create a different overlay for each of the 10 items in that collection based on the index to achieve something similar to this Netflix hub?

![image](https://user-images.githubusercontent.com/8560685/229028045-62058f63-e25e-465f-9866-452dc3aa19fc.jpeg)

**Comment by meisnate12** [comment]
not at the moment
